Webpack cannot import JavaScript via ../ path

## Problem

In CSS frameworks folder such as `bootstrap4/`, `app.js` imports common JavaScript:

```javascript
import * as Common from '../common';
```

where `common/index.js` re-export `common/PageLoader.js` stuff

Travis error is

```
ERROR in ./common/index.js
Module not found: Error: Can't resolve './PageLoader' in '/home/travis/build/Al-un/css/common'
 @ ./common/index.js 2:0-71 3:0-51 3:0-51 3:0-51
 @ ./bootstrap4/app.js
```

--- a/bootstrap4/app.js
+++ b/bootstrap4/app.js
@@ -3,7 +3,7 @@ import '../common/common.scss';
 import './app.scss';
 // Common JavaScript
 // import * as Common from '../common';
-import { onHashChange, loadDefaultPage } from '../common/PageLoader.js';
+import { onHashChange, loadDefaultPage } from './../common/PageLoader.js';
 // Dependent HTML pages
 import './html/home.html';
 import './html/data.html';

--- a/bootstrap4/app.js
+++ b/bootstrap4/app.js
@@ -2,7 +2,7 @@
 import 'common.scss';
 import './app.scss';
 // Common JavaScript
-import { onHashChange, loadDefaultPage } from 'pageLoader.js';
+import { onHashChange, loadDefaultPage } from 'pageLoader';
 // Dependent HTML pages
 import './html/home.html';
 import './html/data.html';

--- a/bootstrap4/app.js
+++ b/bootstrap4/app.js
@@ -1,8 +1,8 @@
 // Styles
-import '../common/common.scss';
+import 'common.scss';
 import './app.scss';
 // Common JavaScript
-import { onHashChange, loadDefaultPage } from '../common/pageLoader.js';
+import { onHashChange, loadDefaultPage } from 'pageLoader.js';
 // Dependent HTML pages
 import './html/home.html';
 import './html/data.html';

--- a/bootstrap4/app.js
+++ b/bootstrap4/app.js
@@ -1,9 +1,8 @@
 // Styles
-import '../common/common.scss';
+import 'common.scss';
 import './app.scss';
 // Common JavaScript
-// import * as Common from '../common';
-import { onHashChange, loadDefaultPage } from './../common/PageLoader.js';
+import { onHashChange, loadDefaultPage } from 'PageLoader.js';
 // Dependent HTML pages
 import './html/home.html';
 import './html/data.html';

--- a/bootstrap4/app.js
+++ b/bootstrap4/app.js
@@ -3,7 +3,7 @@ import '../common/common.scss';
 import './app.scss';
 // Common JavaScript
 // import * as Common from '../common';
-import { onHashChange, loadDefaultPage } from '../common/PageLoader';
+import { onHashChange, loadDefaultPage } from '../common/PageLoader.js';
 // Dependent HTML pages
 import './html/home.html';
 import './html/data.html';

--- a/bootstrap4/app.js
+++ b/bootstrap4/app.js
@@ -3,7 +3,7 @@ import '../common/common.scss';
 import './app.scss';
 // Common JavaScript
 import * as Common from '../common';
-import * as PageLoader from '../common/PageLoader';
+import { onHashChange, loadDefaultPage } from '../common/PageLoader';
 // Dependent HTML pages
 import './html/home.html';
 import './html/data.html';
@@ -12,8 +12,8 @@ import './html/modal.html';
 import './html/notification.html';
 import './html/panel.html';
 
-const onHashChange = async () => {
-  const page = await PageLoader.onHashChange();
+const onBootstrapHashChange = async () => {
+  const page = await onHashChange();
   // Home page specific
   if (page === 'home') {
     document.getElementById('content').classList.remove('container');
@@ -22,5 +22,5 @@ const onHashChange = async () => {
   }
 };
 
-window.onhashchange = onHashChange;
-PageLoader.loadDefaultPage();
+window.onhashchange = onBootstrapHashChange;
+loadDefaultPage();

--- a/bootstrap4/app.js
+++ b/bootstrap4/app.js
@@ -1,8 +1,8 @@
 // Styles
-import 'common.scss';
+import '../common/common.scss';
 import './app.scss';
 // Common JavaScript
-import { onHashChange, loadDefaultPage } from 'PageLoader.js';
+import { onHashChange, loadDefaultPage } from '../common/pageLoader.js';
 // Dependent HTML pages
 import './html/home.html';
 import './html/data.html';

--- a/bootstrap4/app.js
+++ b/bootstrap4/app.js
@@ -2,7 +2,7 @@
 import '../common/common.scss';
 import './app.scss';
 // Common JavaScript
-import * as Common from '../common';
+// import * as Common from '../common';
 import { onHashChange, loadDefaultPage } from '../common/PageLoader';
 // Dependent HTML pages
 import './html/home.html';

--- a/bulma/app.js
+++ b/bulma/app.js
@@ -3,6 +3,7 @@ import '../common/common.scss';
 import './app.scss';
 // Common JavaScript
 import * as Common from '../common';
+import { onHashChange, loadDefaultPage } from '../common/PageLoader';
 // Dependent HTML pages
 import './html/home.html';
 import './html/data.html';
@@ -11,12 +12,8 @@ import './html/modal.html';
 import './html/notification.html';
 import './html/panel.html';
 
-const onHashChange = async () => {
-  await Common.onHashChange();
-};
-
 window.onhashchange = onHashChange;
-Common.loadDefaultPage();
+loadDefaultPage();
 
 /**
  * Bulma specific stuff

--- a/bulma/app.js
+++ b/bulma/app.js
@@ -2,7 +2,7 @@
 import 'common.scss';
 import './app.scss';
 // Common JavaScript
-import * as Common from 'common';
+import * as Common from '../common';
 import { onHashChange, loadDefaultPage } from 'pageLoader';
 // Dependent HTML pages
 import './html/home.html';

--- a/bulma/app.js
+++ b/bulma/app.js
@@ -2,8 +2,8 @@
 import '../common/common.scss';
 import './app.scss';
 // Common JavaScript
-import * as Common from '../common';
-import { onHashChange, loadDefaultPage } from '../common/PageLoader';
+import * as Common from '../common/index.js';
+import { onHashChange, loadDefaultPage } from '../common/PageLoader.js';
 // Dependent HTML pages
 import './html/home.html';
 import './html/data.html';

--- a/bulma/app.js
+++ b/bulma/app.js
@@ -1,9 +1,9 @@
 // Styles
-import '../common/common.scss';
+import 'common.scss';
 import './app.scss';
 // Common JavaScript
-import * as Common from '../common/index.js';
-import { onHashChange, loadDefaultPage } from '../common/PageLoader.js';
+import * as Common from 'common';
+import { onHashChange, loadDefaultPage } from 'pageLoader';
 // Dependent HTML pages
 import './html/home.html';
 import './html/data.html';

--- a/common/index.js
+++ b/common/index.js
@@ -1,7 +1,3 @@
-// ---------- To re-export page loaders
-import { loadDefaultPage, loadPage, onHashChange } from './PageLoader';
-export { loadDefaultPage, loadPage, onHashChange };
-
 // ---------- Hide and seeks
 export const toggleDisplay = targetId => {
   document.getElementById(targetId).classList.toggle('not-displayed');

--- a/raw/app.js
+++ b/raw/app.js
@@ -1,8 +1,9 @@
 // Styles
-import '../common/common.scss';
+import 'common.scss';
 import './app.scss';
 // Common JavaScript
 import * as Common from '../common';
+import { onHashChange, loadDefaultPage } from 'pageLoader';
 // Dependent HTML pages
 import './html/home.html';
 import './html/data.html';
@@ -11,5 +12,5 @@ import './html/modal.html';
 import './html/notification.html';
 import './html/panel.html';
 
-window.onhashchange = Common.onHashChange;
-Common.loadDefaultPage();
+window.onhashchange = onHashChange;
+loadDefaultPage();

--- a/webpack.root-config.js
+++ b/webpack.root-config.js
@@ -29,6 +29,14 @@ function devConfig(framework, options) {
         publicPath: '/',
         library: 'app'
       },
+      resolve: {
+        // https://webpack.js.org/configuration/resolve/#resolve-modules
+        // https://gist.github.com/sokra/27b24881210b56bbaff7#resolving-options
+        modules: [
+          path.resolve(__dirname, 'common'),
+          'node_modules'
+        ]
+      },
       // Building mode
       mode: 'development',
       devtool: 'inline-source-map',

--- a/webpack.root-config.js
+++ b/webpack.root-config.js
@@ -32,9 +32,10 @@ function devConfig(framework, options) {
       resolve: {
         // https://webpack.js.org/configuration/resolve/#resolve-modules
         // https://gist.github.com/sokra/27b24881210b56bbaff7#resolving-options
+        // https://moduscreate.com/blog/es6-es2015-import-no-relative-path-webpack/
         modules: [
           path.resolve(__dirname, 'common'),
-          'node_modules'
+          path.resolve('./node_modules')
         ]
       },
       // Building mode

